src/Umbraco.Core/Services/MemberService.cs

```diff
@@ -941,19 +936,25 @@ public void Delete(IMember member)
         {
             using (var uow = UowProvider.GetUnitOfWork())
             {
-                if (uow.Events.DispatchCancelable(Deleting, this, new DeleteEventArgs<IMember>(member)))
-                {
-                    uow.Commit();
-                    return;
-                }
-
-                var repository = RepositoryFactory.CreateMemberRepository(uow);
-                repository.Delete(member);
+                Delete(uow, member);

```

Comment:
No problem that this got moved out into a method. But on line 940 there is uow.Commit(); not sure is this is an extra one by mistake @zpqrtbnk as you perform a commit after the delete on line 954. But not sure you would be attemptint to commit once this method completes & runs on line 940.

Sorry this is may be how its supposed to work that there is some kind of nested commit happening or similar?!